packages/common/src/feedback/utils.ts/set-display-map-keyword.ts

```diff
@@ -0,0 +1,20 @@
+import { MAP_SURVEY_TYPEKEYWORD } from "../constants";
```

Comment:
The folder containing this file is named `utils.ts` but should probably be named just `utils`.

The `packages/common/index.ts` file does not export the contents from the `packages/common/feedback` dir.

We should probably consider consolidating the existing `surveys` and `feedback` directories into just `feedback`.
1. Move modules in `packages/common/surveys/utils` to `packages/common/feedback/utils` (move tests as necessary too)
2. Move `get-*` & `is-*` modules in `packages/common/surveys` to `packages/common/feedback/utils` (move tests as necessary too)
3. Update `packages/common/feedback/utils/index.ts` to export all modules from that dir
4. Update `packages/common/feedback/index.ts` to export `utils`
5. Update export in `packages/common/index.ts` from `"./surveys"` to `"./feedback"`